### PR TITLE
Corrected Portuguese streams

### PR DESCRIPTION
**Pull request by ksl1989**
* **Caçavision**
  * This is channel about caça, hunting, not caca, poo.
  * Official website shows no space in the name.
  * Added LQ, MQ and HQ to low, medium and hight quality streams, so there aren't three streams with the same name.
  * Added category "Hobby" which seems the most appropriate given the choice, even if I don't like calling it that.

* **TV Fátima**
  * Removing duplicate stream: both streams seem the same, it's just one has the domain name the other IP number. URL is otherwise the same, both .m3u8 can be found both ways. Leaving the one with the domain name because it's more recognizable but changed to the _livestream.m3u8_ file because it includes metadata.
  * Added tvg-id and tvg-name from the provided EPG file.
  * Added category.

* **Porto Canal**
  * Added category.

* **Sobrenatural TV**
  * Added category.
  * Appended "(backup)" to the second stream's name.
* **Kuriakos TV**
  * Added category.
  * Appended "(backup)" to the second stream's name.
* **Kuriakos Kids**
  * Changed category from Kids to Religious

Note: This choice of categories may be controversial. Both Sobrenatural TV and Kuriakos TV are owned by evangelical churches and claim to be general content, family oriented channels, but actually looking at the programming it's clear most of it is selling Jesus or the bible. The point of claiming they're not a religious channel is to proselytise to people who wouldn't usually look at religious channels. If we just take their word and mark it as general we are helping their cause, not being accurate. The music and movies channel are different because these kind of channels often are themed (horror movies, classical music...) so having a music channel with a Christian theme is fine, I think. The Kids channel worries me especially, most people will let children watch kids channels assuming the shows are either entertaining or educational, but this channel also includes a lot of evangelical propaganda directed at children, with the purpose of converting or instilling dogma. I believe it should be marked as religious. Religious people are free to watch it and force it on their children, but people who don't want this shouldn't be led to do it just because the channel was marked as "Kids" TV. I'm explaining myself here because this will apply to other channels and you might not agree, so I'm making my reasoning clear.

* Removed
  * **FM O Dia TV** because it is:
    * Not a TV channel (it seems a web stream from a radio station that isn't even broadcasting any video content except for the logo).
    * Not Portuguese, it's Brazilian.
    * Repeated, it's already on the Brazil list.
  * **TV Maná Córdoba** because it is a religious TV channel from a Portuguese Evangelical church but it is in Spanish and intended for Argentina. It's also repeated, it's already in the Argentina list.
  * **TVI Reality** because it isn't working, at least not for me, and I can watch the official stream which requires a dynamically generated token. The domain is the same, this might just be an old link.

### CODE REVIEW

File: channels/pt.m3u [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
 https://cdn039.fractalmedia.es/hls/caz_pt_low.m3u8
-#EXTINF:-1 tvg-id="Caca Vision PT" tvg-name="Caca Vision PT" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/3qB3JL7.png" group-title="",Caca Vision
+#EXTINF:-1 tvg-id="Caca Vision PT" tvg-name="Caca Vision PT" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/3qB3JL7.png" group-title="Hobby",Caçavision (MQ)
 https://cdn039.fractalmedia.es/hls/caz_pt_med.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/kOnFXO1.png" group-title="",FM O Dia TV
-http://c2901-slbps-sambavideos.akamaized.net/livet/FMoDiaVideo_14d480346fa04ebcb4f03173f2dae707/ngrp:livestream/chunklist_b898304.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/GbQuvWG.png" group-title="Movies",Kuriakos Cine
 http://c2.manasat.com:1935/kcine/kcine3/FluxusTV.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/tfc2BYM.png" group-title="Kids",Kuriakos Kids
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/tfc2BYM.png" group-title="Religious",Kuriakos Kids
```
Comment (freearhey) [inline channels/pt.m3u:12]: Incorrect category

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by freearhey** [comment]
Everything is great except for the category "Kuriakos Kids" channel. Please correct it so I can approve this pull request.

**Comment by ksl1989** [comment]
> Everything is great except for the category "Kuriakos Kids" channel. Please correct it so I can approve this pull request.

I very much disagree a religious channel directed at children should be categorized as "Kids" rather than "Religious", but I did as requested.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Thanks!

**Pull request merged**